borg/archive.py

```diff
@@ -420,6 +409,17 @@ def restore_attrs(self, path, item, symlink=False, fd=None):
                 os.lchflags(path, item[b'bsdflags'])
             except OSError:
                 pass
+        xattrs = item.get(b'xattrs', {})
```

Comment:
2 things we should have:
- a comment here that says it must be done after chown and why
- a unit test that fails before and succeeds after this cs